use-cases.md

```diff
@@ -57,10 +58,12 @@ Point of Sale systems \(POS\) need to be able to create charges in Mews. With Me
 
 ### Inital Setup
 
-The integration should use the [Get all services](operations/services.md#get-all-services) operation to retrieve all services the property has configured in Mews. Once all services are retrieved, the service which you would like all charges to be sent under should be selected. Ideally this would be done via the UI in the setup page. Alternatively, a field for the `ServiceId` to be entered in can be used.
+The integration should use the [Get all services](operations/services.md#get-all-services) operation to retrieve all services the property has configured in Mews. Once all services are retrieved, the service which the property would like all charges to be sent under should be selected. Ideally this would be done via the UI in the setup page.
 
 In the POS system, accounting categories usually exist such as entree, main, dessert, beverage or alcohol. In order for the revenue to be correctly reported in Mews with these categories, they should be correctly mapped against accounting categories in Mews. The POS integration should use the [Get all account categories](operations/finance.md#get-all-accounting-categories) operation to retrieve a list of all `AccountingCategories` which the property has in Mews. These should be mapped via a UI in the POS.
 
+A property may operate different outlets such as a bar, room service and/or multiple restaurants. In Mews, these are referred to as an [Outlet](operations/enterprises.md#get-all-outlets). The POS system should use the [Get all outlets](operations/enterprises.md#get-all-outlets) operation to retrieve all the outlets the property has configured in Mews. These should be mapped via a UI in the POS.
```

Comment:
Here, Outlet links to get all outlets, not the outlet itself. @BrittonA  Pls go through the PR and double check all the links.